pkg/core/pipeline/condition/main.go

```diff
@@ -29,14 +29,13 @@ type Condition struct {
 
 // Config defines conditions input parameters
 type Config struct {
-	resource.ResourceConfig `yaml:",inline"`
+	resource.ResourceConfig `yaml:",inline,omitempty"`
```

Comment:
Why there isn't any type on this line like for ex https://github.com/updatecli/updatecli/pull/671/files#diff-552d39479cf11c87201c084db9cb6ead8c57697852e048e47839addfc64d9400L32-R32?

Reply:
This « form » is a kind of inheritance between structs. It allows the children struct to have the same fields as the parent (parent being the resource.Resource here).
I believe it is named « composition » in Go

Reply:
More references to help you:
- https://go.dev/ref/spec#Struct_types => golang reference
- https://go101.org/article/type-embedding.html

Reply:
Oh, I understand the confusion: this last example is not the best naming 😅

In that case, it is a named field which is expected to hold an object of type `transformer.Transformers`. 
It is not an inheritance of `transformer.Transformers`.